cmd/edv-rest/startcmd/start.go

```diff
@@ -59,6 +61,14 @@ const (
 		" creating or accessing underlying databases." +
 		" Alternatively, this can be set with the following environment variable: " + databasePrefixEnvKey
 
+	databaseTimeoutFlagName      = "database-timeout"
+	databaseTimeoutEnvKey        = "EDV_DATABASE_TIMEOUT"
+	databaseTimeoutFlagShorthand = "o"
+	databaseTimeoutFlagUsage     = "Total time in seconds to wait until the datasource is available before giving up." +
```

Comment:
I think you mean "database" not "datasource"

Reply:
fixed